extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
@beck410 we have been setting `-webkit-overflow-scrolling: touch` on body and other components like carousel for a while now. We are aware and have fixed dome bugs like scroll freeze, etc.. but have note noticed anything with child position relative elements getting broken. Any sample codes that you have will help us test this.